src/plugins/lvm-dbus.c

```diff
@@ -677,6 +684,8 @@ static void call_lvm_method_sync (const gchar *obj, const gchar *intf, const gch
     log_task_status (log_task_id, log_msg);
     g_free (log_msg);
 
+    g_variant_unref (ret);
```

Comment:
We have one extra unref somwhere here (`Warning: g_variant_unref: assertion 'value->ref_count > 0' failed` in the test results) and I think it's this one -- we already have unref calls in all branches above, so it shouldn't be needed here, right?

Reply:
@vojtechtrefny Yes, all branches have unref calls above, so this was an extra. I've updated the commit to remove it. Thanks for the catch!